builders/build/eth-api/dev-env/tenderly.md

```diff
@@ -0,0 +1,103 @@
+3. (Optional) You can give your contract a name
+4. (Optional) You can toggle the **Add more** slider to on if you'd like to add additional contracts. This will allow you to add more contracts after the initial contract has been added
+5. Finally to add the contract to the dashboard, click **Add contract**
+
+![Add a contract](/images/builders/build/eth-api/dev-env/tenderly/tenderly-3.png)
+
+After a contract has been added, it will appear in the list of contracts on the **Contracts** dashboard. If the contract hasn't been verified yet, the dashboard will display an **Unverified** status along with a **Verify** button.
+
+![Contract in list of contracts](/images/builders/build/eth-api/dev-env/tenderly/tenderly-4.png)
+
+To take full advantage of the Tenderly tool set, it is recommended that you verify your smart contracts, which you can do by clicking on **Verify**. You can choose to verify your contract by uploading the contract's JSON, ABI, or source code. For more information, please refer to [Tenderly's documentation](https://docs.tenderly.co/monitoring/smart-contract-verification/verifying-a-smart-contract#verifying-a-smart-contract){target=_blank}.
+
+### Create a Fork {: #fork-moonbeam }
+
+Tenderly's forking feature simulates the live Moonbeam network in an isolated environment, which enables you to interact with deployed contracts and live on-chain data.
+
+Tenderly makes creating a fork through the dashboard quite simple. To get started, click on **Forks** on the left-side menu and then click **Create Fork**. From there, you can take the following steps:
+
+1. Select **Moonbeam** or **Moonriver** from the **Network** dropdown
+2. (Optional) Give your fork a name
+3. If you only need data up until a specific block, you can toggle the **Use Latest Block** slider to off and specify the block number. Otherwise, you can leave the slider as is to include all blocks up until the latest block
+4. Click **Create**
+
+![Fork Moonbeam](/images/builders/build/eth-api/dev-env/tenderly/tenderly-5.png)
+
+Once you've created your fork, you can start using it by deploying a contract to it or creating a transaction simulation using it. 
+
+To deploy a contract to your fork, you can click on the **Deploy Contract** button, upload your contract's source code, and set the compiler configurations. Once you submit the deployment, you'll see the transaction of your deployment appear under the **Simulated Transactions** tab and can click on the simulation for more information.
+
+![Fork simulations](/images/builders/build/eth-api/dev-env/tenderly/tenderly-6.png)
+
+To create additional simulations, you can click the **New Simulation** button and enter in the configurations for the simulation. For more information on how to create a simulation, please refer to [Tenderly's documentation](https://docs.tenderly.co/simulations-and-forks/how-to-simulate-a-transaction){target=_blank}.
```

Comment:
Maybe add an small extra section where you say that Tenderly offers a lot more features than the ones you just describe above?

I feel that closing this page like this might be a bit abrupt